modules/phase_field/include/userobjects/DiscreteNucleationInserter.h

```diff
@@ -43,4 +45,15 @@ class DiscreteNucleationInserter : public DiscreteNucleationInserterBase
 
   /// total nucleation rate
   Real _nucleation_rate;
+
+  /// determine if a fixed radius or variable radius are used
+  const bool _fixed_radius;
+
+  /// contain the value of the nucleus radius
+  const Real _radius;
+
+  /// store the local nucleus radius
+  const MaterialProperty<Real> & _local_radius;
```

Comment:
you can combine those. Users can pass a number instead of a material property name

Reply:
combined